python_src/dead_instrumenter/__init__.py

```diff
@@ -0,0 +1,28 @@
+from .instrumenter import (
+    InstrumentedProgram,
+    find_all_markers,
+    get_instrumenter,
+    annotate_with_static,
+)
+
+from diopter.compiler import (
```

Comment:
Does it really make sense to import all of these by default? 

Shouldn't `__init__.py` import the bare minimum user facing stuff?

In this case, `InstrumentedProgram` and `instrument_program`. (And `annotate_with_static` but it should be dropped from this repo as it's a bit irrelevant)